Antidote/DialingCallViewController.m

```diff
@@ -0,0 +1,159 @@
+//
+//  DialingCallViewController.m
+//  Antidote
+//
+//  Created by Chuong Vu on 7/1/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import "DialingCallViewController.h"
+#import "Masonry.h"
+#import "Helper.h"
+#import "OCTCall.h"
+#import "OCTChat.h"
+#import "OCTSubmanagerCalls.h"
+
+static const CGFloat kIndent = 50.0;
+
+@interface DialingCallViewController () <RBQFetchedResultsControllerDelegate>
+
+@property (strong, nonatomic) RBQFetchedResultsController *callController;
+
+@property (strong, nonatomic) UILabel *nameLabel;
+@property (strong, nonatomic) UIButton *cancelCallButton;
+@property (strong, nonatomic) UILabel *timerLabel;
+
+@end
+
+@implementation DialingCallViewController
+
+#pragma mark - Life cycle
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+
+    [self createNameLabel];
+
+    [self createEndCallButton];
+    [self createCallTimer];
+
+    [self installConstraints];
+}
+
+#pragma mark -  RBQFetchedResultsControllerDelegate
+
+- (void)controllerDidChangeContent:(RBQFetchedResultsController *)controller
```

Comment:
It would be better to move this method into Abstract controller and add method `didUpdateCall` I was talking about.